widget/embedded/src/Widget.tsx

```diff
@@ -40,6 +43,13 @@ export type WidgetProps = {
   config?: WidgetConfig;
 };
 
+export type WidgetStateProps = {
+  providers: WidgetConfig['wallets'];
```

Comment:
`providers`, `options` and `onUpdateState` isn't available inside the `config`? I'm trying to reduce the props.

Reply:
fixed.